plugins/app-backend/src/service/router.ts

```diff
@@ -132,8 +133,12 @@ export async function createRouter(
   const staticRouter = Router();
   staticRouter.use(
     express.static(resolvePath(appDistDir, 'static'), {
-      setHeaders: res => {
-        res.setHeader('Cache-Control', CACHE_CONTROL_MAX_CACHE);
+      setHeaders: (res, path) => {
+        if (path === injectedConfigPath) {
+          res.setHeader('Cache-Control', 'no-cache');
```

Comment:
Tbh I don't think we should go for this short-term fix. The config tends to be in the main chunk, which also almost always ends up being fairly large. I think it's better to aim for a proper fix straight away since this is gonna have a fairly large impact on page loading time.

Reply:
Just to be clear, the `no-cache` directive doesn't prevent the browser to reuse the cached content, when the ETAG is the same as the previous one, and the server returns a 304 (NOT_MODIFIED) status code.

Mozilla docs:

> **no-cache**
>
> The `no-cache` response directive indicates that the response can be stored in caches, but the response must be validated with the origin server before each reuse, even when the cache is disconnected from the origin server.
>
> ...
>
> If you want caches to always check for content updates while reusing stored content, `no-cache` is the directive to use. It does this by requiring caches to revalidate each request with the origin server.
>
> Note that `no-cache` does not mean "don't cache". `no-cache` allows caches to store a response but requires them to revalidate it before reuse. If the sense of "don't cache" that you want is actually "don't store", then `no-store` is the directive to use.

So I don't think it's a problem to add this directive on a Javascript file, even if it's the main chuck, knowing that the content would not returned if not modified.

I might be missing something, but that was my reasoning why this PR should be a valuable short-term fix while waiting for a wider refactoring of config injection (which will take much more time).

Reply:
Ah alright, this seems like a good fix for now then 👍. Could you extract it to a constant similar to the other header values? Something like `CACHE_CONTROL_REVALIDATE_CACHE`.

Should we set `max-age` too?

Reply:
> Should we set max-age too?

I don't think so, afaict.

Reply:
> Could you extract it to a constant similar to the other header values?

Done in commit 0a659c9b1db68c299ffbde9b788ae4b455eacce5